backend/src/ee/services/scim/scim-service.ts

```diff
@@ -386,17 +420,19 @@ export const scimServiceFactory = ({
     });
 
     if (!active) {
-      await deleteOrgMembership({
+      await deleteOrgMembershipFn({
```

Comment:
Should ideally also be a transaction

Reply:
The function actually already is a transaction; the transaction is inside of it.